plugins/azure/resoto_plugin_azure/resource/base.py

```diff
@@ -539,6 +566,16 @@ def add_node(self, node: AzureResourceType, source: Optional[Json] = None) -> Op
             # add edge from subscription to resource
             last_edge_key = self.add_edge(self.subscription, node=node)
 
+        # create provider link
+        if node._metadata.get("provider_link") is None:
+            link_tpl = "https://portal.azure.com/#@/resource/subscriptions{resource_id}/overview"
+            try:
+                src = source.copy() if source else {}
+                src["resource_id"] = node.id
+                node._metadata["provider_link"] = link_tpl.format(**src)
+            except Exception as e:
+                log.warning(f"Can not compute provider_link for {node} with template: {link_tpl}: {e}")
```

Comment:
why so complicated? The only parameter is id.
```suggestion
            node._metadata["provider_link"] = f"https://portal.azure.com/#@/resource/subscriptions{node.id}/overview"
```

Reply:
Thanks.